packages/blade/src/components/Collapsible/_decisions/decisions.md

```diff
@@ -0,0 +1,186 @@
+# Collapsible
+
+Collapsible is used to allow users to toggle the visibility of hidden content within a container.
+
+This document outlines the API of `Collapsible` component. This shares similarities with [Accordion's API](https://github.com/razorpay/blade/pull/1189).
+
+<div align="center">
+  <img src="./collapsible-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Collapsible](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-629874&t=sVxH3DOnx3L3F9rO-0)
+
+## Interactions
+
+Collapsible shares the opening / closing interaction with Accordion:
+
+### Collapsible button
+
+https://github.com/razorpay/blade/assets/6682655/e37daea8-4cbb-42df-b33b-4d32dc0eab65
+
+### Collapsible link
+
+https://github.com/razorpay/blade/assets/6682655/eec42564-4a43-4449-96e1-7c81639d60e7
+
+## API
+
+Annotated component with props ([figma](https://www.figma.com/file/cX625jYMAX1Roq8hLifFrn/Collapsible-breakdown?type=design&node-id=0-1&t=2fpp9pyvcuMxVTjW-0)):
+
+![Collapsible breakdown](./collapsible-breakdown.png)
+
+```jsx
+import { Collapsible, CollapsibleButton, CollapsibleLink, CollapsibleBody } from '@razorpay/blade';
+
+<Collapsible>
+  <CollapsibleButton>Button</CollapsibleButton>
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+
+<Collapsible>
+  <CollapsibleLink>Button</CollapsibleLink>
+  <CollapsibleBody>
+    <Box>Slot</Box>
+  </CollapsibleBody>
+</Collapsible>;
+```
+
+> **Note**
+>
+> - `Collapsible` will compose (`CollapsibleButton` or `CollapsibleLink`), `CollapsibleBody`. This ensures all interactions, a11y and animations work seamlessly at user end.
+> - `CollapsibleButton` and `CollapsibleLink` will support a subset of relevant props from `Button` and `Link` respectively.
+
+### Collapsible
+
+| Prop              | Type            | Default     | Description                                                        | Required |
+| ----------------- | --------------- | ----------- | ------------------------------------------------------------------ | -------- |
+| children          | `JSX`           | `undefined` | Composes `CollapsibleButton`, `CollapsibleLink`, `CollapsibleBody` | ✅       |
+| direction         | `bottom`, `top` | `bottom`    | Direction in which the content expands                             |          |
```

Comment:
Should we call this `placement` to match it with Tooltip?

Reply:
I think `direction` fits better with the interaction (which direction does the content expand in vs which placement does the content expand in) 😅

Reply:
That is true. I think `placement` makes sense on Tooltip because it is not with respect to trigger. Trigger isn't part of the component. So it "places" itself. Whereas here `direction` makes sense because we're talking about it in the context of trigger 😅 

Like if we were to put this prop on `CollapsibleBody` then `placement` would've made sense because you're "placing" it wrt to the trigger and main component.

I think we can go with direction in that case.